exercises/concept/rpn-calculator/.docs/instructions.md

```diff
@@ -1,6 +1,6 @@
 # Instructions
 
-While working at _Instruments of Texas_, you are tasked to work on an experimental calculator written in Elixir. Your team is having a problem with some operations raising errors and crashing the process. You have been tasked to write a function which wraps the operation function so that the errors can be handled more elegantly with idiomatic Elixir code.
+While working at _Instruments of Texas_, you are tasked to work on an experimental Reverse Polish Notation calculator written in Elixir. Your team is having a problem with some operations raising errors and crashing the process. You have been tasked to write a function which wraps the operation function so that the errors can be handled more elegantly with idiomatic Elixir code.
```

Comment:
This is the first exercise in the "series", so it should mention that RPN = Reverse Polish Notation, not everyone will know that.

Reply:
```suggestion
While working at _Instruments of Texas_, you are tasked to work on an experimental Reverse Polish Notation [RPN] calculator written in Elixir. Your team is having a problem with some operations raising errors and crashing the process. You have been tasked to write a function which wraps the operation function so that the errors can be handled more elegantly with idiomatic Elixir code.
```